src/UnitsOfMeasure.Extra.fsproj

```diff
@@ -1,11 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Version>1.0.0</Version>
-    <PackageVersion>1.0.0-beta-005</PackageVersion>
+    <PackageVersion>1.0.0-beta-006</PackageVersion>
```

Comment:
The build script checks wether the version here and in RELEASE_NOTES are out-of-sync to to know when it should push a new release, so you don't need to update this manually :) But I can revert just this line now.

Reply:
Ah, makes sense (in https://github.com/jet/jet.jsonnet.converters et al, I use MinVer, which works well wrt inferring a suitable version, but having it tie in with release notes is more important than that).

Feel free to ping if you want me to expand on this (to be clear though: it seems the script and process is already better as-is and if it aligns with processes in sister projects it would be crazy to foist the dependency for something which will hopefully go into maintenance mode very soon!)

Thanks for the speedy merge!